### PR TITLE
Fix files to process env var

### PR DESCRIPTION
**Pull request by joel-wright**

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -45,4 +45,4 @@ def _convert_to_bool(x: str) -> bool:
     os.getenv("TARGET_LANGUAGES", "en").lower().split(",")
 )  # comma-separated 2-letter ISO codes
 ENCODER_SUPPORTED_LANGUAGES: Set[str] = {"en"}
-FILES_TO_PARSE: str = os.getenv("files_to_parse")
```
Comment (THOR300) [inline src/config.py:48]: Below is the definition of env vars in the pulumi job flow. Therefore, I believe the original os.getenv("files_to_parse") would be correct here?

"ContainerOverrides": {
                                        "Environment": [
                                            {
                                                "Name": "files_to_parse",
                                                "Value.$": "$.ObjectDetails.tasks_string",
                                            },
                                        ]
                                    },

File: cli/run.sh [hunk 2]
```diff
@@ -1 +1,4 @@
-python -m cli.text2embeddings --s3 $EMBEDDINGS_INPUT_PREFIX $INDEXER_INPUT_PREFIX --device=cpu
\ No newline at end of file
+#!/bin/bash
+set -e
```
Comment (THOR300) [inline cli/run.sh:2]: Out of interest is this best practice or is this necessary as we were successfully passing in the prefixes prior to this 'set -e' change?

Comment (joel-wright) [reply cli/run.sh:2]: those 2 lines are currently "best practice & not really necessary", but as soon as another line turns up in this script we'll be happy they're there.

File: src/config.py (hunk 1 above)
Comment (joel-wright) [reply src/config.py:48]: are you sure you're not looking at the parser stage?

Comment (joel-wright) [reply src/config.py:48]: we should change the name of this env var to avoid confusion

Comment (THOR300) [reply src/config.py:48]: Ahh yep you're right it caps!

**Review by THOR300 — APPROVED** [review]

**Pull request merged**